src/client/java/teammates/client/scripts/sql/VerifyNonCourseEntityCounts.java

```diff
@@ -0,0 +1,99 @@
+package teammates.client.scripts.sql;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.cloud.datastore.QueryResults;
+import com.googlecode.objectify.cmd.Query;
+
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Root;
+
+import teammates.client.connector.DatastoreClient;
+import teammates.client.util.ClientProperties;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.entity.Account;
+import teammates.storage.entity.BaseEntity;
+
+public class VerifyNonCourseEntityCounts extends DatastoreClient {
+    private VerifyNonCourseEntityCounts() {
+        String connectionUrl = ClientProperties.SCRIPT_API_URL;
+        String username = ClientProperties.SCRIPT_API_NAME;
+        String password = ClientProperties.SCRIPT_API_PASSWORD;
+
+        HibernateUtil.buildSessionFactory(connectionUrl, username, password);
+    }
+
+    public static void main(String[] args) throws Exception {
+        new VerifyNonCourseEntityCounts().doOperationRemotely();
+    }
+
+    private void printEntityVerification(String className, int datastoreCount, long psqlCount) {
+        System.out.println("========================================");
+        System.out.println(className);
+        System.out.println("Objectify count: " + datastoreCount);
+        System.out.println("Postgres count: " + psqlCount);
+        System.out.println("Correct number of rows?: " + (datastoreCount == psqlCount));
+    }
+
+
+    private Long countPostgresEntities(Class<? extends teammates.storage.sqlentity.BaseEntity> entity) {
+        HibernateUtil.beginTransaction();
+        CriteriaBuilder cb = HibernateUtil.getCriteriaBuilder();
+        CriteriaQuery<Long> cr = cb.createQuery(Long.class);
+        Root<? extends teammates.storage.sqlentity.BaseEntity> root = cr.from(entity);
+
+        cr.select(cb.count(root));
+        
+        
+        Long count = HibernateUtil.createQuery(cr).getSingleResult();
+        HibernateUtil.commitTransaction();
+        return count;
+    }
+
+    private void verifyReadNotification() {
+        Query<Account> accountQuery = ofy().load().type(Account.class);
+        QueryResults<Account> iterator = accountQuery.iterator();
+
+        int datastoreReadNotifications = 0;
```

Comment:
Should this count be of type `long` as well?

Reply:
This can be but I'm standardizing all objectify query counts to be int since, the aggregation `count()` function for objectify returns an int and not long

Reply:
I see! For thes non Course entities I think it is fine to keep it int as the size should not be that big